src/pages/Details.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <section class="page">
+        <header class="page__header" :style="{ backgroundImage: `url(${backgroundImage})` }">
+            <Button :active="true" v-on:click="onClickHome">HOME</Button>
+        </header>
+        <main class="page__body">
+            <Movie
+                :id="movie.id"
+                :name="movie.name"
+                :genre="movie.genre"
+                :year="movie.year"
+                :rating="movie.rating"
+                :image="movie.image"
+                :description="movie.description"
+            />
+        </main>
+    </section>
+</template>
+
+<script>
+    import { mapState, mapActions } from 'vuex';
+    import backgroundImage from '../assets/bg.png';
+    import Movie from '../components/Movie.vue'
+    import Button from '../components/Button.vue';
+    export default {
+        name: 'Details',
+        components: {
+            Movie,
+            Button
+        },
+        data: function () {
+            return {
+                backgroundImage: backgroundImage,
+            }
+        },
+        methods: {
+            ...mapActions(['fetchMovie']),
+            onClickHome: function () {
+                this.$router.push({ name: 'home' })
```

Comment:
sam here, you can assign routes names to the constants

Reply:
ok, thanks